app/models/financialDetails/ChargeItem.scala

```diff
@@ -140,11 +140,10 @@ object ChargeItem {
 
   implicit val format: Format[ChargeItem] = Json.format[ChargeItem]
 
-  def filterChargeWithOutstandingAmount(isChargeTypeEnabled:Boolean, chargeType: ChargeType*)
-                  (chargeItem: TransactionItem): Boolean = {
+  def filterAllowedCharges(isChargeTypeEnabled:Boolean, chargeType: ChargeType*)
```

Comment:
as I understand we don't have any ITs coverage for filtering?

Reply:
No it doesn't look like it, will add some